src/components/Proposal/Create/PluginForms/ResetFormButton.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import * as css from "./ResetFormButton.scss";
+
+export default (props: {
+  resetToDefaults: () => void,
+  isSubmitting: boolean,
+}): JSX.Element => {
+  return (
+    <button className={css.resetFormButton} type="button" disabled={props.isSubmitting}
+    // eslint-disable-next-line react/jsx-no-bind
+      onClick={(): void => {
```

Comment:
Is it possible to call this function not inline? If you can declare it in the class scope and just call it. Better for readability and no need to disable eslint manually.

Reply:
Done